WebKitDev/Functions/Install-FromMsi.ps1

```diff
@@ -49,7 +49,16 @@ function Install-FromMsi {
     Write-Information -MessageData ('Installing {0} ...' -f $name) -InformationAction Continue;
     Write-Information -MessageData ('msiexec {0}' -f ($msiArgs -join ' ')) -InformationAction Continue;
 
-    Start-Process msiexec -Wait -ArgumentList $msiArgs;
+    # According to https://stackoverflow.com/a/23797762 caching the handle is required to get ExitCode
+    $process = Start-Process msiexec -PassThru -ArgumentList $msiArgs;
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUserDeclaredVarsMoreThanAssignments', '', Scope='Function')]
+    $handle = $process.Handle;
+    $process.WaitForExit();
+
+    if ($process.ExitCode -ne 0) {
+        Write-Error ('{0} installer failed with exit code {1}' -f $name, $process.ExitCode);
```

Comment:
Do we want to continue in this case? My understanding was Write-Error is not meant to be fata.

Reply:
Added a stop to it